src/Sulu/Bundle/SecurityBundle/Exception/RoleKeyAlreadyExistsException.php

```diff
@@ -12,31 +12,31 @@
 namespace Sulu\Bundle\SecurityBundle\Exception;
 
 /**
- * Exception is thrown when a Role is created or updated with an already existing name.
+ * Exception is thrown when a Role is created or updated with an already existing key.
  */
-class RoleNameAlreadyExistsException extends \Exception
```

Comment:
we should just deprecate this one

Reply:
if we keep uniquiness we should just keep or introduce a `RoleAlreadyExistsException`.